app/src/main/res/layout/activity_main.xml

```diff
@@ -1,9 +1,16 @@
 <?xml version="1.0" encoding="utf-8"?>
+<layout>
 <androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
che niazi be in `ConstraintLayout` bironi hast?
benazaram estefade i ham nemishe

Reply:
Roo main activity kar nakardam faghat khastam fragment ro neshon bedam natije ro bebinam

Reply:
hich iradi nadare
bi zahmat eslahesh kon